backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -23,6 +38,10 @@ def refund_reasons
         @refund_reasons ||= RefundReason.active.all
       end
 
+      def refund_params
+        params.require(:refund).permit!
```

Comment:
Any reason not to use a whitelist here, rather than permitting everything?

Reply:
This is done throughout the admin